### ISSUE TITLE
Adding arbitrary boards to favorite list.

### ISSUE THREAD

**Issue by CodeMan99** [waffleio/waffle.io#1840]
I am not a contributor to waffle.io, but I would love to be able to visit this board without needed to hit the "Ask a question" button.

Would be useful too for projects that you often report tickets against, but are not considered a contributor.

I would like see this functionality on the "add boards view" as well as a button on any board I'm view that I have not yet added to my favorites list.

**Comment by jonbrenner** [comment]
Agreed. I want to be able to add a shortcut for an arbitrary board to my favorites list so that I don't have to navigate to it by URL. The favorites list can include just a basic filter toggle to search for projects that have a board set up already:

![include external filter](http://i.imgur.com/vLWBSqx.png)
